### PR TITLE
Prebuild ibc tokens

### PR DESCRIPTION
**Pull request by banshee**
Don't merge this until we merge https://github.com/Sifchain/sifnode/pull/1808

### CODE REVIEW

File: smart-contracts/src/ibcMatchingTokens.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import fs from "fs";
+import * as hardhat from "hardhat";
+import web3 from "web3";
+
+interface TokenAddress {
+    address: string
+}
+
+async function attachIbcToken(bridgeBank: BridgeBank, token: IbcToken) {
+    await token.transferOwnership(bridgeBank.address)
```
Comment (banshee) [inline smart-contracts/src/ibcMatchingTokens.ts:12]: transferOwnership is no longer valid, this needs to be removed

**Comment by banshee** [comment]
> Don't merge this until we merge #1808

That has been merged.

File: smart-contracts/hardhat.config.ts [hunk 2]
```diff
@@ -70,7 +73,15 @@ const config: HardhatUserConfig = {
...
+    },
+    etherscan: {
+        // Your API key for Etherscan
+        // Obtain one at https://etherscan.io/
+        apiKey: process.env['ETHERSCAN_API_KEY']
+      }    
 }
 
+// verify command: 
+// https://prod.liveshare.vsengsaas.visualstudio.com/join?84826512BD698D80121925FFF8F6A0D17A2A
```
Comment (ElliotFriedman) [inline smart-contracts/hardhat.config.ts:85]: Let's remove this.

**Review by ElliotFriedman — CHANGES_REQUESTED** [review]

File: smart-contracts/hardhat.config.ts (hunk 2 above)
Comment (banshee) [reply smart-contracts/hardhat.config.ts]: Done

**Review by ElliotFriedman — APPROVED** [review]

**Pull request merged**